MapboxAndroidDemo/src/global/play/release-notes/en-US/default.txt

```diff
@@ -1 +1 @@
-This release provides several bug fixes and improvements to existing examples
\ No newline at end of file
+This release provides several bug fixes
```

Comment:
Due to changelog there are some new examples, perhaps mention it here.

Reply:
Sure